exec/jdbc-all/pom.xml

```diff
@@ -516,7 +518,7 @@
                   This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
 
                   </message>
-                  <maxsize>40000000</maxsize>
+                  <maxsize>41600000</maxsize>
```

Comment:
How about this one ?

Reply:
Sorry, overlooked it. There is no need more in this increase.